extension/src/experiments/model/status/collect.test.ts

```diff
@@ -14,28 +18,31 @@ describe('collectColoredStatus', () => {
     return mockExperiments
   }
 
-  const collectedColoredStatus = (experiments: Experiment[]) =>
-    collectColoredStatus(
+  it('should set unseen checkpoints and experiments to unselected', () => {
```

Comment:
[F] This is a big behaviour change which affects a lot of tests. I have left it included in this PR because breaking it out would have led to changing a lot of the same tests twice.